lib/ansiblelint/utils.py

```diff
@@ -68,9 +68,12 @@ def initialize_logger(level=0):
     handler = logging.StreamHandler()
     formatter = logging.Formatter('%(levelname)-8s %(message)s')
     handler.setFormatter(formatter)
-    _logger.addHandler(handler)
+    logger = logging.getLogger("ansiblelint")
```

Comment:
How about we don't hardcode the name?
```suggestion
    logger = logging.getLogger(__package__)
```